docs/astrocut/file_formats.rst

```diff
@@ -158,7 +176,24 @@ POS_CORR2    0
 
 The ``TIME`` column is formed by taking the average of the ``TSTART`` and ``TSTOP`` values
 from the corresponding FFI for each row. The ``QUALITY`` column is taken from the ``DQUALITY``
-image keyword in the individual FFI files.
+image keyword in the individual SPOC FFI files.
+
+For cutouts created from TICA cubes, the ``TIMECORR`` column has been removed from the
+PIXELS BinTableHDU. Similar to cutouts made from SPOC cubes, the other columns (aside from
+the ``TIMECORR`` column) present in Mission pipeline TPFs are present in cutouts created
+from TICA cubes, but do not all contain data. The columns that are empty in Astrocut TICA TPFs are:
+
+============ ====================================================
+Column       Value
+============ ====================================================
+RAW_CNTS     -1 filled array in cutout shape
+FLUX_ERR     0 filled array in cutout shape
+FLUX_BKG     0 filled array in cutout shape
+FLUX_BKG_ERR 0 filled array in cutout shape
+QUALITY      0
```

Comment:
`QUALITY` is non-zero right?: https://github.com/spacetelescope/astrocut/blob/061cd6bf5ad7a89630dc304301d0bc4d2d2968fb/astrocut/cube_cut.py#L747

Reply:
For TICA, the `QUAL_BIT` keyword (which is used to populate the `QUALITY` flags in the cutouts) is just a placeholder keyword, and no quality flags were ever implemented in the TICA pipeline.